packages/e2e-tests/scripts/save-examples.ts

```diff
@@ -181,6 +181,12 @@ const knownExamples: TestExampleFile[] = [
     category: "browser",
     runtime: "chromium",
   },
+  {
+    filename: "doc_rr_basic_chromium.html",
+    folder: config.browserExamplesPath,
+    category: "browser",
+    runtime: "chromium",
+  },
```

Comment:
I'd like to land #10182 before this (which will obviate the need for this awkward duplication).